src/docs/integrations/discord/index.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Discord Integration"
+---
+
+## Create a Discord bot
+
+To use Sentry's Discord integration you'll need to first create a Discord bot. Go to the [Discord Developer Portal](https://discord.com/developers/applications) and click __New Application__.
+
+![Create bot](./create_bot.png)
+
+After creating your Discord bot, you'll need to grab a few credentials here. You can find them on the following pages: 
+
+- *Application ID* and *Public key* are located in __General Information__.
+- *Client secret* is located in __OAuth2__ > __General__.
+- *Bot token* is located in __Bot__.
+
+Copy each of these and paste them into <code>config.yml</code> as follows:
+
+```yaml
+discord.application-id: "<application id>"
+discord.public-key: "<public key>"
+discord.client-secret: "<client secret>"
+discord.bot-token: "<bot token>"
+```
+
+**NOTE**: If you're doing local Sentry development, you should use a tool like Ngrok and add `system.url-prefix: "https://<your_subdomain>.ngrok.io"` to your `~/.sentry/config.yml`, otherwise, installing the Discord bot won't work as it will not be able to communicate with your local Sentry instance.
```

Comment:
```suggestion
**NOTE**: If you're doing local Sentry development, you should use a tool like Ngrok and add `system.url-prefix: "https://<your_subdomain>.ngrok.io"` to your `~/.sentry/config.yml`. Otherwise, installing the Discord bot won't work, as it will not be able to communicate with your local Sentry instance.
```